phpBB/index.php

```diff
@@ -420,7 +420,7 @@
 								$last_post = $lang['No_Posts'];
 							}
 
-							if ( count($forum_moderators[$forum_id]) > 0 )
+							if ( count($forum_moderators[$forum_id] ?? $forum_moderators ) > 0 )
```

Comment:
Would this work instead? :
`        if (isset($forum_moderators[$forum_id]) && count($forum_moderators[$forum_id]) > 0)`
Using `$forum_moderators` as the default argument for `count()`when the forum_id is missing isn't very intuitive.

Reply:
True story, sorry haha